docs/environment-variables.md

```diff
@@ -12,6 +12,8 @@ If true, set secure session cookies.
 
 ### MEMCACHED\_HOST
 
+- **Example**: `localhost:11211`
```

Comment:
👍 

We should discuss bringing the docs up to (and improving on) the standard of my.ubuntu.com